src/qtism/data/AssessmentTest.php

```diff
@@ -430,17 +432,207 @@ public function isExclusivelyLinear()
     }
 
     /**
-	 * Whether the AssessmentTest as a TimeLimits component bound to it.
-	 *
-	 * @return boolean
-	 */
+     * Whether the AssessmentTest as a TimeLimits component bound to it.
+     *
+     * @return boolean
+     */
     public function hasTimeLimits()
     {
         return $this->getTimeLimits() !== null;
     }
-    
+
     public function __clone()
     {
         $this->setObservers(new SplObjectStorage());
     }
+
+    /**
+     * Returns an array with all possible paths for a AssessmentTest.
+     * @param asArray true to return an array, false to return an AssessmentItemRefCollection.
+     * @return array of array of AssessmentItemRef or an array AssessmentItemRefCollection, depending on $asArray.
+     * @throws Exception if branching is recursive of backward.
```

Comment:
Please use PHP FQCNs (Full Qualified Class Name). In this example \Exception just to show that it is part of the root namespace.